libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -409,7 +409,7 @@ else if (record.getBoolean(LOCALLY_UPDATED))
         Map<String, Object> fields = new HashMap<String, Object>();
         if (action == Action.create || action == Action.update) {
             for (String fieldName : fieldlist) {
-                if (!fieldName.equals(Constants.ID)) {
+                if (!fieldName.equals(Constants.ID) && !fieldName.equals(Constants.LAST_MODIFIED_DATE)) {
```

Comment:
Being more forgiving.
Devs will likely pull lastModifiedDate for resync. That way, they don't need to exclude it from fieldlist on save.